com.unity.render-pipelines.universal/ShaderLibrary/Debug/DebuggingCommon.hlsl

```diff
@@ -0,0 +1,249 @@
+
+#ifndef UNIVERSAL_DEBUGGING_COMMON_INCLUDED
+#define UNIVERSAL_DEBUGGING_COMMON_INCLUDED
+
+#include "Packages/com.unity.render-pipelines.universal/ShaderLibrary/Debug/DebugViewEnums.cs.hlsl"
+
+// Set of colors that should still provide contrast for the Color-blind
+#define kPurpleColor float4(156.0 / 255.0, 79.0 / 255.0, 255.0 / 255.0, 1.0) // #9C4FFF
+#define kRedColor float4(203.0 / 255.0, 48.0 / 255.0, 34.0 / 255.0, 1.0) // #CB3022
+#define kGreenColor float4(8.0 / 255.0, 215.0 / 255.0, 139.0 / 255.0, 1.0) // #08D78B
+#define kYellowGreenColor float4(151.0 / 255.0, 209.0 / 255.0, 61.0 / 255.0, 1.0) // #97D13D
+#define kBlueColor float4(75.0 / 255.0, 146.0 / 255.0, 243.0 / 255.0, 1.0) // #4B92F3
+#define kOrangeBrownColor float4(219.0 / 255.0, 119.0 / 255.0, 59.0 / 255.0, 1.0) // #4B92F3
+#define kGrayColor float4(174.0 / 255.0, 174.0 / 255.0, 174.0 / 255.0, 1.0) // #AEAEAE
```

Comment:
Following up with @TheoWong-pixel on this. Similar stuff is also present on HDRP, so we could try and figure out a shared solution for both, which might be something like this:
1. get UX to produce a list of 12 color-blind-friendly gradient-like colors
2. add them to Editor Preferences -> Colors (or if it seems odd to put 12 new colors there just for this, we could put them in a RP settings asset instead, or perhaps use an actual gradient)
3. once colors are available, put them in a constant buffer and use them for both HDRP & URP.

Whatever we decide, I think this should be done in a separate PR because there's various dependencies which means it will take time.